scrapers/Pornditos.yml

```diff
@@ -19,13 +24,44 @@ xPathScrapers:
         selector: //div[@id="js-small-thumb"]/img/@src
         postProcess:
           - replace:
-              - regex: "http://sm.members.khcdn.com/shoots/([^/]+)/([^/]+)/rollover.*"
-                with: "http://x-images5.bangbros.com/$1/shoots/$2/members/450x340.jpg"
+              - regex: 'http://sm\.members\.khcdn\.com/shoots/([^/]+)/([^/]+)/rollover.*'
+                with: "http://x-images5.bangbros.com/$1/shoots/$2/big_trailer.jpg"
+              - regex: '5\.bangbros\.com/cuchimami'
+                with: "4.bangbros.com/cuchimami"
       Date:
         selector: $info/p[starts-with(text(),"Added to")]/text()[last()]
         postProcess:
           - replace:
               - regex: on\s
                 with:
           - parseDate: Jan 2, 2006
-# Last Updated August 08, 2021
+
+  culionerosScraper:
+    common:
+      $info: //h2[@class="h2-trailer-block"]
+    scene:
+      Title: $info//span[@class="h2-trailer-style"]/text()
+      Details: 
+        selector: //p[@class="para-class t-para"]/text() 
+        postProcess:
+          - replace:
+              - regex: \n
+                with: " "
```

Comment:
I tried a few scenes without the pp and it seemed fine.
Do you have an example(url) of a scene that needs the pp ? (`\n` should have been removed by stash anyway)

Reply:
yeah, I was having trouble getting it to recognize the line breaks from this link specifically
`http://en.culioneros.com/videos/carol-finds-a-guy-on-the-street-to-fuck-in-public_102934.htm`